model_analyzer/reports/report_manager.py

```diff
@@ -467,63 +495,96 @@ def _build_detailed_table(self, model_config_name):
                 row = [
-                    model_config.get_field('name'),
-                    model_config.dynamic_batching_string(), instance_group_str,
+                    measurement.perf_config()['concurrency-range'],
                     measurement.get_metric('perf_latency').value(),
-                    measurement.get_metric('perf_client_response_wait').value(),
+                    measurement.get_metric(
+                        'perf_client_response_wait').value(),
                     measurement.get_metric('perf_server_queue').value(),
-                    measurement.get_metric('perf_server_compute_input').value(),
-                    measurement.get_metric('perf_server_compute_infer').value(),
+                    measurement.get_metric(
+                        'perf_server_compute_input').value(),
+                    measurement.get_metric(
+                        'perf_server_compute_infer').value(),
                     measurement.get_metric('perf_throughput').value(),
                     measurement.get_metric('cpu_used_ram').value()
                 ]
                 detailed_table.insert_row_by_index(row)
         return detailed_table
 
-    def _get_gpu_stats(self, report_key):
+    def _build_detailed_info(self, model_config_name):
         """
-        Gets names and memory infos of GPUs used in best measurements
+        Constructs important info sentence about the model config
+        specified
         """
 
-        gpu_names = []
-        max_memories = []
-        seen_gpus = set()
-        cpu_only = True
+        model_config, measurements = self._detailed_report_data[
+            model_config_name]
+        instance_group_string = model_config.instance_group_string()
+        dynamic_batching_string = model_config.dynamic_batching_string()
+        platform = model_config.get_field('platform')
+
+        # Measurements and GPU info
+        if model_config.cpu_only():
+            sentence = (
+                f"The model config {model_config_name} requests {instance_group_string} "
+                f"instances. {len(measurements)} measurements were obtained for the model config "
+                f"on CPU with platfrom {platform}. ")
+        else:
+            gpu_dict = self._get_gpu_stats(measurements=measurements)
+            gpu_names = ','.join(list(gpu_dict.keys()))
+            max_memories = ','.join(
+                [str(x) + ' GB' for x in gpu_dict.values()])
+            sentence = (
+                f"The model config {model_config_name} requests {instance_group_string} "
+                f"instances. {len(measurements)} measurements were obtained for the model config "
+                f"on GPU(s) {gpu_names} with memory limit(s) {max_memories}. This model "
+                f"uses the platform {platform}. ")
+
+        # Dynamic batching
+        if dynamic_batching_string == 'N/A':
+            sentence += "This model does not support batching. "
```

Comment:
How do we know that it doesn't support dynamic batching? Is it for the case that `max_batch_size=0`?

Reply:
Yes exactly